package.json

```diff
@@ -4,17 +4,24 @@
   "private": true,
   "license": "Apache-2.0",
   "dependencies": {
+    "@fortawesome/fontawesome": "^1.1.4",
+    "@fortawesome/fontawesome-free-regular": "^5.0.8",
+    "@fortawesome/fontawesome-free-solid": "^5.0.8",
```

Comment:
Would be nice to record how all of these increase main.js, though we should eventually automate this (https://github.com/rahafoundation/raha.io/issues/45)